crates/ln-dlc-node/src/ln/coordinator_event_handler.rs

```diff
@@ -422,10 +428,15 @@ where
         );
     }
 
-    // TODO(holzeis): check liquidity option here
-    let channel_value = expected_outbound_amount_msat / 1000 * crate::LIQUIDITY_MULTIPLIER;
+    let expected_outbound_amount =
+        Decimal::from(expected_outbound_amount_msat) / Decimal::from(1000);
+    let trade_up_to = Decimal::from(liquidity_request.trade_up_to);
+    let coordinator_leverage =
+        Decimal::try_from(liquidity_request.coordinator_leverage).expect("to fit into decimal");
 
     let fake_scid = FakeScid::new(requested_next_hop_scid);
+    let channel_value = expected_outbound_amount + (trade_up_to / coordinator_leverage);
+    let channel_value = channel_value.to_u64().expect("to fit into u64");
```

Comment:
:wrench: It would be good to extract these mathsy bits into functions that can be documented and even tested.